scripts/zones/Kazham/npcs/Magriffon.lua

```diff
@@ -1,25 +1,49 @@
 -----------------------------------
 -- Area: Kazham
 -- NPC: Magriffon
--- Standard Info NPC
+-- Involved in Quest: Gullible's Travels, Even More Gullible's Travels,
+-- Location: (I-7)
 -----------------------------------
 
 package.loaded["scripts/zones/Kazham/TextIDs"] = nil;
+require("scripts/globals/settings");
+require("scripts/globals/quests");
+require("scripts/globals/titles");
 require("scripts/zones/Kazham/TextIDs");
 
+
 -----------------------------------
 -- onTrade Action
 -----------------------------------
 
 function onTrade(player,npc,trade)
-end; 
+    if(player:getQuestStatus(OUTLANDS, GULLIBLES_TRAVELS) == QUEST_ACCEPTED) then
+        if(trade:getGil() >= player:getVar("MAGRIFFON_GIL_REQUEST")) then
+            player:startEvent(0x0092);

```

Comment:
I told him to use confirmTrade because tradeComplete will delete the gil
you traded (instead of just the amount you owed)

On Tue, Jul 14, 2015 at 5:36 PM, whasf <notifications@github.com> wrote:

> In scripts/zones/Kazham/npcs/Magriffon.lua
> <https://github.com/DarkstarProject/darkstar/pull/1718#discussion_r34632616>
> :
>
> >  -----------------------------------
> >  -- onTrade Action
> >  -----------------------------------
> >
> >  function onTrade(player,npc,trade)
> > -end;
> > +    if(player:getQuestStatus(OUTLANDS, GULLIBLES_TRAVELS) == QUEST_ACCEPTED) then
> > +        if(trade:getGil() >= player:getVar("MAGRIFFON_GIL_REQUEST")) then
> > +            player:startEvent(0x0092);
>
> Maybe tradeComplete() here? I think we've had problems in the past trying
> to finish a trade across functions/events
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/DarkstarProject/darkstar/pull/1718/files#r34632616>.
>